fe/src/main/java/org/apache/doris/alter/Alter.java

```diff
@@ -118,30 +118,30 @@ public void processDropMaterializedView(DropMaterializedViewStmt stmt) throws Dd
             ErrorReport.reportDdlException(ErrorCode.ERR_BAD_DB_ERROR, dbName);
         }
 
-        db.writeLock();
+        String tableName = stmt.getTableName().getTbl();
+        Table table = db.getTable(tableName);
```

Comment:
Could use `getTableOrThrowException `

Reply:
Here should throw userException or try and catch then throw ddl exception because we need some detailed error message?